Source/ablastr/math/fft/WrapCuFFT.cpp

```diff
@@ -69,10 +71,12 @@ namespace ablastr::math::anyfft
 
     void DestroyPlan(FFTplan& fft_plan)
     {
+        ABLASTR_PROFILE("ablastr::math::anyfft::DestroyPlan", true);
         cufftDestroy( fft_plan.m_plan );
     }
 
     void Execute(FFTplan& fft_plan){
+        ABLASTR_PROFILE("ablastr::math::anyfft::Execute", true);
```

Comment:
Please use `false` for device synchronize. We luckily have a new AMReX runtime option for this and can thus avoid unnecessary device syncs (costly).